Integrations/GitHub/GitHub.py

```diff
@@ -0,0 +1,277 @@
+                str(r['total_count']) + ' found issues.\nComplete list of issues: ' + str(not r['incomplete_results']),
+                results, ['ID', 'title', 'state', 'link'])
+        print_results['EntryContext'] = {
+            'GitHub.Searches(val.query==obj.query)': {'query': query, 'results': results}
+        }
+    else:
+        print_results['HumanReadable'] = 'No issues found for this query!'
+    demisto.results(print_results)
+    return r
+
+
+def count_total_downloads():
+    r = http_request('GET', 'repos/teizenman/Demisto-start/releases')
+    output = []
+    for asset in r[0]['assets']:
+        count = makehash()
+        count['asset_name'] = asset['name']
+        count['download_count'] = asset['download_count']
+        output.append(count)
+    demisto.results({
+        'Type': entryTypes['note'],
+        'ContentsFormat': formats['markdown'],
+        'Contents': r,
+        'HumanReadable': tableToMarkdown('Download counts:', output, ['asset_name', 'download_count']),
+        'EntryContext': {
+            'GitHub.Download-counts(val.asset_name==obj.asset_name)': output
+        }
+    })
+    return r
+
+
+''' EXECUTION '''
+LOG('command is %s' % (demisto.command(),))
+try:
+    if demisto.command() == 'list-all-issues':
+        list_all_issues()
+
+    elif demisto.command() == 'create-issue':
+        title = demisto.args().get('title')
+        body = demisto.args().get('body')
+        assignees = argToList(demisto.args().get('assignees'))
+        labels = argToList(demisto.args().get('labels'))
+        create_issue(title, body, assignees, labels)
+
+    elif demisto.command() == 'edit-issue':
+        issue_num = demisto.args().get('issue_num')
+        title = demisto.args().get('title')
+        body = demisto.args().get('body')
+        assignees = argToList(demisto.args().get('assignees'))
+        labels = argToList(demisto.args().get('labels'))
+        edit_issue(issue_num, title, body, assignees, labels)
+
+    elif demisto.command() == 'close-issue':
+        issue_num = demisto.args().get('issue_num')
+        close_issue(issue_num, 'close')
+
+    elif demisto.command() == 'make-search':
+        search_query = demisto.args().get('search_query')
+        sort_by = demisto.args().get('sort_by')
+        order_by = demisto.args().get('order_by')
+        make_search(search_query, sort_by, order_by)
+
+    elif demisto.command() == 'count-downloads':
+        count_total_downloads()
+
+    elif demisto.command() == 'test-module':
+        list_all_issues()  # result code 200
+        create_issue('From Demisto', 'Right from the integration window', ['teizenman'],
+                     ['good first issue'])  # result code 201
+        edit_issue(3, 'From Demisto', 'Right from the integration window', ['teizenman'],
+                   ['enhancment'])  # result code 200
+        close_issue(3, 'close')  # result code 200
+        demisto.results('ok')
+except Exception, e:
```

Comment:
Your version only supports python2, we want to write new code in python3 compatibility 
```suggestion
except Exception as e:
```